testing/ef-tests/README.md

```diff
@@ -0,0 +1,12 @@
+## Run [ethereum/consensus-spec-tests](https://github.com/ethereum/consensus-spec-tests)
+
+
+Run Tests this will automatically download test data
+```bash
+make test
+```
+
+Clean test files
+```bash
+make clean
+```
```

Comment:
In general, I think this README could use a fair amount more useful information....
- outline of the crate structure
- what kinds of datatypes are being tested?
- how to add tests in future forks & make sure that they're tested
- any other info you think would be useful to someone who just looks at this crate, and doesn't want to go digging through the code to understand what's going on

Reply:
Thank you for the feedback I will update the readme :saluting_face: